### PR TITLE
fix bugs the proposal system testing

### PR DESCRIPTION
**Pull request by Maliaotw**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [x] **Bugfix**

## Description
1. change the description to 1000 characters the description
2. Hide tutorial in modification step
3. Word wrap for proposal & review comment

### CODE REVIEW

**Comment by Maliaotw** [comment]
[Fix ALL the bugs found during the proposal system testing](https://trello.com/c/j0ezAODh/24-fix-all-the-bugs-found-during-the-proposal-system-testing)

File: src/locale/en_US/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -4305,7 +4305,7 @@ msgstr "biography"
 msgid ""
 "Describe yourself with 500 characters or less. There will be no formatting."
```
Comment (mattwang44) [inline src/locale/en_US/LC_MESSAGES/django.po:4306]: Though your changes make the users see what we want them to see, the message id still shows the old information. Please also change:

- the help text in `src/users/models.py`
- the message id in the `src/locale/{en_US, zh_Hant}/LC_MESSAGES/django.po`

File: src/static/css/pages/_proposals.scss [hunk 2]
```diff
@@ -107,6 +107,7 @@ body.dashboard{
                 }
                 .simple-field, .text-field, .markdown-field {
                     line-height: 150%;
+									  overflow-wrap: anywhere;
```
Comment (mattwang44) [inline src/static/css/pages/_proposals.scss:110]: Please replace the tab with space for consistency.

```suggestion
                    overflow-wrap: anywhere;
```

File: src/core/templatetags/pycontw_tools.py [hunk 3]
```diff
@@ -18,3 +17,8 @@ def get_path_category(url):
     if not result:
         return 'unmatched'
     return result.groupdict().get('category', 'uncategorized')
+
+@register.simple_tag
+def get_model_verbose_name_raw(obj):
+    print('obj',obj._meta.verbose_name_raw)
```
Comment (mattwang44) [inline src/core/templatetags/pycontw_tools.py:23]: I believe this line is for debugging. Please delete it.

File: src/templates/default/proposals/_proposal_update.html [hunk 4]
```diff
@@ -26,11 +26,15 @@ <h3 class="dashboard-header">
 				{% trans 'Proposal' %}
 			</a>
 		</li>
-		<li role="presentation">
-			<a href="#reviews" aria-controls="reviews" role="tab" data-toggle="tab">
-				{% trans 'Reviews' %}
-			</a>
-		</li>
+		{% get_model_verbose_name_raw object as obj_verbose_name %}
+		{% if 'tutorial proposal' != obj_verbose_name %}
```
Comment (mattwang44) [inline src/templates/default/proposals/_proposal_update.html:30]: nit: change the sequence for readability

```suggestion
		{% if obj_verbose_name != 'tutorial proposal' %}
```

**Review by mattwang44 — CHANGES_REQUESTED** [review]
As we discussed before, developers are encouraged to keep their [commits small and atomic](https://www.freshconsulting.com/atomic-commits/). Please survey how to use `git rebase` command to divide the commit that consisted of three purposes into three individual commits.

**Pull request closed without merging**